torch/nn/utils/_stateless.py

```diff
@@ -43,3 +44,39 @@ def functional_call(module, parameters_and_buffers, args, kwargs=None):
         else:
             out = module(args, **kwargs)
     return out
+
+def per_sample_call(module, batch_size, args, kwargs=None):
```

Comment:
Location seems fine for now (since we're not ready to go beta in pytorch 1.11) but long term this should probably go into torch.nn.utils?

Reply:
Moved to `_per_sample_grad.py` just so it doesn't get too messy in `_stateless` and added a comment to that function--basically because this depends on `_stateless.function_call` which imports `torch.nn.Module` we get a circular dependency if we try to expose this in `torch.nn.utils`